pkg/utils/helper.go

```diff
@@ -132,28 +131,6 @@ func GetHostName() (string, error) {
 	return hostName.HostName, nil
 }
 
-// GetAPIServerFQDN gets the API Server FQDN from the kubeconfig file
-func GetAPIServerFQDN() (string, error) {
```

Comment:
💡 Same as above comment, that we could move this to `netwrokbound_collector.go` but completely removing which include taking small chunk of log out of the logs which gets dumb - looks ok to my eyes but need folks like @JunSun17 or @rzhang628, @qpetraroia to verify what are the key reasons it was there at first place.